bindings/experimental/distrdf/python/DistRDF/ComputationGraphGenerator.py

```diff
@@ -56,8 +57,8 @@ def get_action_nodes(self, node_py=None):
             # current DistRDF node as the head node
             node_py = self.headnode
         else:
-            if (node_py.operation.is_action() or
-                    node_py.operation.is_instant_action()):
+            if isinstance(node_py.operation, (Action, InstantAction)):
+                # TODO: remove the isinstance call
```

Comment:
How do you plan to remove it?

Reply:
In the PR for systematic variations I'm already splitting the `generate_computation_graph` in multiple functions with `singledispatch` depending on the type of the input node. Since this code is touched in that commit I thought to leave it out for now. On second thought, it might be better to do it already in this PR